tests/integration/conftest.py

```diff
@@ -46,6 +50,74 @@ def testing_api_key(environ: str) -> str:
     return os.environ["LABELBOX_TEST_API_KEY_STAGING"]
 
 
+def beta_endpoint(fn):
+
+    def beta(client, *args, **kwargs):
+        try:
+            client.endpoint = client.endpoint.replace("/graphql", "/_gql")
+            return fn(client, *args, **kwargs)
+        finally:
+            client.endpoint = client.endpoint.replace("/_gql", "/graphql")
+
+    return beta
+
+
+@beta_endpoint
+def cancel_invite(client, invite_id):
+    """
+    Do not use. Only for testing.
+    """
+    query_str = """mutation CancelInvitePyApi($where: WhereUniqueIdInput!) {
+            cancelInvite(where: $where) {id}}"""
+    client.execute(query_str, {'where': {'id': invite_id}})
+
+
+@beta_endpoint
+def get_project_invites(client, project_id):
+    """
+    Do not use. Only for testing.
+    """
+    id_param = "projectId"
+    query_str = """query GetProjectInvitationsPyApi($from: ID, $first: PageSize, $%s: ID!) {
+        project(where: {id: $%s}) {id
+        invites(from: $from, first: $first) { nodes { %s
+        projectInvites { projectId projectRoleName } } nextCursor}}}
+    """ % (id_param, id_param, results_query_part(Invite))
+    return list(
```

Comment:
why use `list` here?

Reply:
I want to fetch all results before the result is returned. Otherwise this query won't work since it will be outside of the experimental scope.

Reply:
Removed.